cmd/controller/apps_reconciler.go

```diff
@@ -5,16 +5,24 @@ package controller
 
 import (
 	"github.com/go-logr/logr"
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
 	kcclient "github.com/vmware-tanzu/carvel-kapp-controller/pkg/client/clientset/versioned"
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/reftracker"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	secret    = "secret"
```

Comment:
seems weird to have these as constants.

Reply:
Intention here was to make sure consistent kind is being used for keys to avoid using strings directly. But can remove and use regular strings instead.

Reply:
it seems like it's only being used in one place.

Reply:
Yeah, I could move it somewhere more central, but maybe let's just remove for now and use strings.